conf/documentation.conf

```diff
@@ -1607,6 +1607,13 @@ description=<<EOT
 Treat OCSP response errors as 'soft' failures and still accept the certificate.
 EOT
 
+[radius_configuration.forward_key_balanced]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Send in the proxy request the attribute PacketFence-KeyBalanced to randomize the load balance in the next PacketFence RADIUS server
```

Comment:
Can you just add a brief description to explain the benefit of using that attribute like:
"Used to ensure requests are balanced when they come from @lzammit's sandwich"